manifest/xyz.jvyden/QuickPick/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "QuickPick",
+	"id": "xyz.jvyden.QuickPick",
+	"description": "A ResoniteModLoader mod that lets you quickly pick ProtoFlux nodes and components in desktop mode.",
```

Comment:
'A ResoniteModLoader mod that lets you...' is not really needed here. Something more like 'Quickly pick ProtoFlux nodes and components in desktop mode with CTRL+G' could be more descriptive and includes mentioning the keybind to use it.